src/particles/transformation/CoordinateTransformation.cpp

```diff
@@ -9,8 +9,8 @@
  */
 #include "CoordinateTransformation.H"
 
-#include "T2Z.H"
-#include "Z2T.H"
+#include "TO_FIXED_S.H"
+#include "TO_FIXED_T.H"
```

Comment:
Let's call these
```suggestion
#include "ToFixedS.H"
#include "ToFixedT.H"
```
for consistency with the rest of the style? :)